### ISSUE TITLE
Conflict with cargo readme

### ISSUE THREAD

**Issue by xoac** [ashleygwilliams/cargo-generate#212]
[cargo readme](https://github.com/livioribeiro/cargo-readme) use similar syntax like cargo generate it cause a problem to have a template that contains `README.tpl` file with for example `{{badges}}`.

Example error:
```
➜ cargo generate --git ../github.com/xoac/crates-io-lib-template  --name crates-io-lib-template-test
 Creating project called `crates-io-lib-template-test`...
Error:  Error replacing placeholders `/home/sr/projects/tmp/crates-io-lib-template-test/README.tpl`
Info: caused by liquid: Unknown variable
  with:
    requested variable=badges
    available variables=authors, crate_name, project-name
```

Liquid support [raw](https://shopify.github.io/liquid/tags/raw/)

**Comment by k0pernicus** [comment]
Hi, thanks for reporting the issue.

Did you know that you can use a `cargo-generate.toml` to exclude the file `README.tpl` if you don't want to touch it?

In this case, the file will be excluded when parsing all tags, and the file will remains the same at the end :)

**Comment by k0pernicus** [comment]
The explanation is given in the README, "Include / Exclude" section.  
To use this feature, you just have to create a `cargo-generate.toml` file in your project, which the content:

```toml
[template]
exclude = ["README.tpl"] # or "*.tpl" if you want to excluse all your template files...
```

**Comment by xoac** [comment]
Thank for asnwer.

I want to use `README.tpl` check it on my template [crates-io-template](https://github.com/xoac/crates-io-lib-template).

As you can see there is [raw tag](https://github.com/xoac/crates-io-lib-template/blob/master/README.tpl#L4) that allow cargo-generate and cargo-readme work together - but require #213 to work.

**Comment by sassman** [comment]
@xoac I have tried your template that you mentioned. With the code of PR #225 and there was no error. 

The README.tpl contains that `{% raw %}` tag, and it seems that it correctly processed. Like below:

```
[![crates.io](https://img.shields.io/crates/v/cool-stuff.svg)](https://crates.io/crates/cool-stuff)
[![Documentation](https://docs.rs/cool-stuff/badge.svg)](https://docs.rs/cool-stuff/)
![CI master](GITHUB_ORG_PATHcool-stuff/workflows/Continuous%20integration/badge.svg?branch=master)
{{badges}}

# {{crate}}
...
```

Could you quick check if that is your expected result?

**Comment by xoac** [comment]
For me the correct way to check this would be include test (from #213)  in your  proposed PR.

The changes https://github.com/ashleygwilliams/cargo-generate/pull/225/files#diff-ab5bd5cb636a34f0984ce86c80edbd7fR15 should resolve my issue.

**Comment by sassman** [comment]
@xoac Exactly. That is what I did, I took that code and the template of yours and that succeeded. But I guess you can tell better, if everything works as expected.

**Comment by xoac** [comment]
I mean If you copy test from my PR and   then run  `cargo test` without failure - then it's ok.

**Comment by sassman** [comment]
@xoac ok, I'll have a look at this particular test then.

**Comment by sassman** [comment]
@xoac thanks for pointing me to that test. I've included it in #225 and it's passing. So it should be all good.

**Comment by xoac** [comment]
Closed with https://github.com/ashleygwilliams/cargo-generate/pull/225 thanks @k0pernicus
